packages/EasyCfhighlander/src/Console/Commands/AbstractTemplatesCommand.php

```diff
@@ -377,6 +383,24 @@ protected function getTemplateName(string $template): string
         return \sprintf('%s/%s.twig', $this->getTemplatePrefix(), $template);
     }
 
+    /**
+     * Determine if the .easy directory exists
+     *
+     * @param string $cwd
+     *
+     * @return string
+     */
+    private function getEasyDirectory(string $cwd): string
+    {
+        // If easy-docker* file already exists in cwd, .easy directory will not be used/created
```

Comment:
Please make sure it looks for the parameters for cfhighlander 😛

Reply:
Fixed